test/unit/ssz_ex_test.exs

```diff
@@ -465,8 +465,8 @@ defmodule Unit.SSZExTest do
 
   test "serialize and deserialize bitlist" do
     encoded_bytes = <<160, 92, 1>>
-    assert {:ok, decoded_bytes} = SszEx.decode(encoded_bytes, {:bitlist, 16})
-    assert {:ok, ^encoded_bytes} = SszEx.encode(decoded_bytes, {:bitlist, 16})
+    assert {:ok, decoded_bytes} = SszEx.decode(encoded_bytes, {:bitlist, 30})
```

Comment:
Why this change?

Reply:
Good finds, thanks for checking! I undid this change, It wasn't necessary. It was something that I was testing manually and forgot to change back.